src/register.c

```diff
@@ -1153,8 +1166,8 @@ op_yank(oparg_T *oap, int deleting, int mess)
 #if defined(FEAT_CLIPBOARD) && defined(FEAT_X11)
     int			did_star = FALSE;
 #endif
+    int			i;
 
-				    // check for read-only register
```

Comment:
Why is this comment removed? A similar comment still exists before `valid_yank_reg()` in `stuff_yank()`.

Reply:
I got a bit trigger happy with 'dd'.